projects/styles/taiga-ui-global.less

```diff
@@ -1,10 +1,7 @@
-@import 'taiga-ui-local.less';
```

Comment:
Some style-files (for example, `tui-form.less`) below uses constants from this file.

Can we really delete it ?